src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/PerformerNameAndAddressTest.java

```diff
@@ -27,10 +27,10 @@ void when_edifactStringDoesNotStartWithCorrectKey_expect_illegalArgumentExceptio
     }
 
     @Test
-    void when_edifactStringIsPassed_expect_returnAPerformingOrganisationNameAndAddressObject() {
-        assertThat(performingOrganisationNameAndAddress)
+    void when_edifactStringIsPassed_expect_returnAPerformingOrganizationNameAndAddressObject() {
+        assertThat(performingOrganizationNameAndAddress)
             .usingRecursiveComparison()
-            .isEqualTo(PerformerNameAndAddress.fromString("NAD+SLA+++LONDON CITY HOSPITAL"));
+            .isEqualTo(PerformerNameAndAddress.fromString("NAD+SLA+++ST JAMES?'S UNIVERSITY HOSPITAL"));
```

Comment:
My usual comments about having to leave the test method to understand why we expect this.

Also, this structure is reversed: `assertThat(expected).isEqualTo(actual)`, should be `assertThat(actual).isEqualTo(expected)`

Reply:
`performingOrganizationNameAndAddress` is a variable at the top of the test class, and is used in a number of tests here. It wouldn't make sense to copy+paste that same value in different tests.
Scrolling up a bit to see the test data > copy+pasting extra code

Reply:
I'll be arguing in my presentation next week that actually it does make sense 😁  and scrolling up a bit < copy+pasting (with the caveat that variations in the test data is also a good thing).

Not a blocker for me